build/docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+# Copyright 2020 Intel Corporation
+# SPDX-License-Identifier: Apache 2.0 
+
+version: "2.4"
+
+services:
+
+  iot_platform_builder:
+    build:
+      context: .
+      dockerfile: Dockerfile
+      args:
+        use_remote : ${use_remote}
+    image: iot_platform_builder
+    container_name: iot_platform_builder
+    restart: on-failure:5
+    # Mapping to the user workspace of 'sdouser' as created in the Dockerfile.
+    # If the username is updated in the Dockerfile, please replace 'sdouser' with the same.
+    volumes:
+      - ./../../iot-platform-sdk:/home/sdouser/iot-platform-sdk:rw
+    environment:
+      - http_proxy=${http_proxy}
+      - https_proxy=${https_proxy}
+      - no_proxy=${no_proxy}
+      - use_remote=${use_remote}
```

Comment:
Add a README.md file that documents how to use the build script (for both local as well as remote build) and what is expected after the build is over.

Reply:
README.md file will be added.